pyproject.toml

```diff
@@ -90,7 +91,8 @@ no-docstring-rgx="^(_|test_|Meta$)"
 disable = """,
     line-too-long,
     bad-continuation,
-    too-few-public-methods
+    too-few-public-methods,
+    arguments-differ,
```

Comment:
What failures is this addressing? Are you confident that they're noise and not legitimate issues?